README.md

```diff
@@ -35,12 +39,10 @@ Please see Gemfile for more details.
 
 - Install the appropriate versions of Ruby, Rails etc.
 
-For example, you can follow [these instructions](https://gorails.com/setup/ubuntu/20.04) if you are using Ubuntu 20.04.
+    For example, you can follow [these instructions](https://gorails.com/setup/ubuntu/20.04) if you are using Ubuntu 20.04.
 
 - Make a git clone of the code in this repo
 
-- Run `gem install pg` to set up the PostgreSQL gem
```

Comment:
Muss ich in der Installationsanleitung angeben, welche zusätzlichen gems installiert werden sollen? Sollten sie ein bundle install durchführen?

Reply:
ja, ich würde schreiben, dass bundle install ausgeführt werden muss. Die einzelnen gems oben aufzulisten, ist nicht notwendig. Die stehen ja im gemfile.